plugin/core/sessions.py

```diff
@@ -386,6 +388,31 @@ def additional_variables(cls) -> Optional[Dict[str, str]]:
         """
         return None
 
+    @classmethod
+    def storage_path(cls) -> str:
+        """
+        The storage path. Use this as your base directory to install server files. Its path is '$DATA/Package Storage'.
```

Comment:
We're referring to `$DATA` in a couple of places but this variable is something that LSP came up with and it doesn't really exist. I'm not sure how clear it is to people reading this code.

Reply:
I'll clarify this in the docstrings